.gitlab/functional_test/system_probe.yml

```diff
@@ -31,6 +31,7 @@ kitchen_test_system_probe_linux_x64:
   before_script:
     - cd $DD_AGENT_TESTING_DIR
     - bash -l tasks/kitchen_setup.sh
+    - source /root/.bashrc && conda activate ddpy3 && conda install openssl gnutls
```

Comment:
Unless there is a very good reason, we should not modify the conda environments on the Gitlab jobs, this should be modified over at https://github.com/DataDog/datadog-agent-buildimages. 

If these changes are not related to the Conda environment we prefer installing via the package manager

Reply:
@nplanel can you install the packages some other way?